lib/yardstick/v2_client/grant_status.rb

```diff
@@ -0,0 +1,36 @@
+module Yardstick
+  module V2Client
+    class GrantStatus
+      include RemoteModel
+
+      attr_accessor :response # TODO: refactor this up so it's available to other models
+
+      resource_uri '/v2/grant_status'
+
+      attr_accessor :writable, :exam_url, :status, :user_id, :id
+
+      alias_method :writable?, :writable
+
+      def update
+        self.response = put(instance_uri, body: {
+          token: token,
+          grant_status: {
+            status: status
+          }
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.